tutorials/36_Building_Fallbacks_with_Conditional_Routing.ipynb

```diff
@@ -1,551 +1,571 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.            "condition": "{{'no_answer' in replies[0].content}}",
</pre><p><code>no_answer' in replies[0].text</code>,the same for the other condition as well  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>